### ISSUE TITLE
<Updating the site's nav bar>

### ISSUE THREAD

**Issue by kshitij01042002** [aritroCoder/IITP-Gymkhana#19]
Original STC Webite
![image](https://user-images.githubusercontent.com/91470808/206702900-d7a8a08f-4e87-49cd-83da-17a0b4c6eccd.png)

New Website
![image](https://user-images.githubusercontent.com/91470808/206702937-b66bb671-8973-4ea5-8629-53c1769f6ee2.png)

Make the site's nav-bar similar to that of the original website

**Comment by Anurag-Deo** [comment]
Actually, you have to change it, but not like the STC website but like this
![image](https://user-images.githubusercontent.com/92918449/206727787-ff4b65fb-358d-489b-a906-7e36ff6f7e0f.png)
The background should be solid black as it is. Each of these will connect to the respective section of the webpage. Except the office bearer, which will be on another page.

**Comment by Anurag-Deo** [comment]
![logo](https://user-images.githubusercontent.com/92918449/206728635-4c2a73ba-ea16-4ead-910f-b7515244ce37.png)
This image can be used in place of arcade one

**Comment by kshitij01042002** [comment]
Okay! I can work on this assign me this issue

**Comment by kshitij01042002** [comment]
Site Navigation bar working perfectly I have attached the content which was available. Other than that it is dummy data from the template itself.

https://user-images.githubusercontent.com/91470808/206781984-5c21d51f-8f85-457c-b618-c2cbf5879739.mp4


In addition to the given issue I have given comments to each section and divison. So that anybody working on this project in future can understamd the code base easily and mosfiy it.
![image](https://user-images.githubusercontent.com/91470808/206782192-96054466-6f0c-4f4f-ad7b-7c2555cb0bc0.png)

If there are any  changes to be made, revert me

![image](https://user-images.githubusercontent.com/91470808/206787339-8a2c7c1f-1b42-4bf0-97cf-2d669c8a9406.png)
PS: Added logo after recording
